NuGetPackageVerifier.json

```diff
@@ -17,16 +17,15 @@
       "DefaultCompositeRule"
     ],
     "packages": {
-      "Microsoft.AspNetCore.Razor.Design": {
+      "Microsoft.NET.Sdk.Razor": {
         "exclusions": {
           "ASSEMBLY_DESCRIPTION": {
             "tools/Microsoft.CodeAnalysis.CSharp.dll": "This assembly is not owned by us and does not follow our conventions.",
             "tools/Microsoft.CodeAnalysis.dll": "This assembly is not owned by us and does not follow our conventions."
           },
           "NEUTRAL_RESOURCES_LANGUAGE": {
             "tools/Microsoft.CodeAnalysis.CSharp.dll": "This assembly is not owned by us and does not follow our conventions.",
-            "tools/Microsoft.CodeAnalysis.dll": "This assembly is not owned by us and does not follow our conventions.",
-            "tools/Newtonsoft.Json.dll": "This assembly is not owned by us and does not follow our conventions."
+            "tools/Microsoft.CodeAnalysis.dll": "This assembly is not owned by us and does not follow our conventions."
```

Comment:
I would bet that if we're putting these things in the SDK, that there will be a desire from that team to avoid shipping duplicates of these assemblies. Is your assumption that we'd handle that during the part where they ingest our bits?

Reply:
Sure, I'll do a follow item for this.